interfaces/apparmor/template.go

```diff
@@ -233,13 +233,15 @@ var defaultTemplate = []byte(`
 
   # Writable home area for this version.
   owner @{HOME}/snap/@{SNAP_NAME}/@{SNAP_REVISION}/** wl,
+  owner @{HOME}/snap/@{SNAP_NAME}/shared/** wl,
```

Comment:
Although I'd like to sync up with everyone on this, the approach looks nice and simple.

One initial comment: shared is perhaps not the best word. It also means "make it public" (share with whom?), and it may imply concurrent use which will never be the case here (only the active snap will touch it). As a preliminary strawman, I'd suggest "common".